fb-webworker.js

```diff
@@ -0,0 +1,109 @@
+importScripts('https://cdn.jsdelivr.net/pyodide/v0.23.4/full/pyodide.js');
+
+const EXCLUDE_CHECKS = [
```

Comment:
Shouldn't all these checks in `EXCLUDE_CHECKS` also be mentioned in fontbakery.js `EXCUSES`?

Reply:
No, these checks are not even run because they crash the program. So no excuses needed...